### DISCUSSION TITLE
Will Prism9 include support for IDialogService on MAUI?

### DISCUSSION THREAD

**Discussion by MartinRobins** [PrismLibrary/Prism#3005] [.NET MAUI]
According to the documentation [here](https://prismlibrary.com/docs/maui/dialogs/mvvm-dialogs.html), Prism does not support the IDialogService on MAUI. Will this be supported when v9 is released?

I have tried using IDialogService with the latest pre-release (9.0.271-pre) and it would appear that the code is there however I have been unsuccessful in displaying anything to date. I don't know if this is because it remains unsupported or simply because I do not yet understand how it works (it appears to have changed since the documentation for XF was produced).

Thanks.

**Comment by dansiegel** [comment]
The docs are not up to date for Prism 9. Thanks for mentioning that. I actually forgot that they weren't previously implemented. But yes Dialogs are implemented in Prism 9 in the currently available preview on NuGet.org.

**Reply by MartinRobins** [reply]
Thanks @dansiegel; guess I need to raise another discussion to see if somebody can tell me what I am doing wrong. I can see the mask, but I cannot see any dialog content!
